src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -66,6 +66,13 @@ lazy-loading iframes could lead to 2-3% median data savings, 1-2% [First
 Contentful Paint](/fcp/) reductions at the median, and 2% [First Input
 Delay](/fid/) (FID) improvements at the 95th percentile.
 
+Additionally, lazy-loading iframes can impart benefits to [Largst Contentful Paint (LCP)](/lcp/).
```

Comment:
Fixed typo, and suggest add `off-screen` as on-screen iframes CAN be LCP elements.

```suggestion
Additionally, lazy-loading off-screen iframes can impart benefits to [Largest Contentful Paint (LCP)](/lcp/).
```

Now in theory, if deep linking an off-screen iframe COULD be LCP element some of the time, but that's same with images and think it's an edge case.